README.md

```diff
@@ -11,30 +11,47 @@ extensions work, please see the following:
 
 ## Compilation / Usage
 
-To setup our extension, run the following (assuming Linux):
+First, you'll need to set up and build the [core Bareflank Hypervisor](https://github.com/bareflank/hypervisor),
+including [its SDK](https://github.com/bareflank/bfsdk).
```

Comment:
Please remove the reference to the SDK as the core hypervisor does this for you, and I don't want to confuse them into thinking they have to do both.